shell/platform/android/external_view_embedder/external_view_embedder.cc

```diff
@@ -73,6 +75,16 @@ void AndroidExternalViewEmbedder::SubmitFrame(
     return;
   }
 
+  auto submit_info = frame->submit_info();
+  int64_t vsync_id = submit_info.vsync_id;
+  if (vsync_id) {
```

Comment:
An integer isn't a boolean: replace this with vsync_id != 0

Reply:
Done